TODO

```diff
@@ -12,7 +12,7 @@ Get output images only produced in BINARY_DIR.
 -- Describe Modularization as part of a ITKv3->ITKv4 transition appendix
 -- Update Acknowledgements
 -- Test clean builds
```

Comment:
@jhlegarreta we should clean up this TODO, as many/most things have been done by now.

Reply:
@dzenanz I agree the list needs to be cleaned up, each item be checked, and maybe issues be opened instead of keeping this file. Not sure if I will have the bandwidth in the short-, mid-term to do this though.

Reply:
Turning the entire file into issues is a great idea! We don't have to address them soon.